fido_metadata.gemspec

```diff
@@ -32,6 +32,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.3"
 
   spec.add_dependency "jwt", "~> 2.0"
+  spec.add_development_dependency "base64", ">= 0.1.0"
```

Comment:
We are actually [using `Base64` in our codebase](https://github.com/cedarcode/fido_metadata/blob/e9402085aaca7f8d003ad06b7fc5408011628ed1/lib/fido_metadata/x5c_key_finder.rb#L34) so we should either add it as a runtime dependency our inline its implementation. I think I'd simply add it as a dependency here. What do you think?